quarkus-workshop-super-heroes/docs/src/docs/asciidoc/0-introduction/introduction-installing.adoc

```diff
@@ -25,6 +27,7 @@ kubectl version # If you used Rancher Desktop and plan to use Kubernetes
 endif::use-local-kubernetes[]
 ----
 
-We have not mentioned cURL up to now, because most modern operating systems ship
-with it pre-installed. If your system is missing cURL,
-the appendix includes xref:appendix-installing-curl[instructions for installing cURL].
+We have not mentioned cURL up to now, because most modern operating systems ship with it pre-installed.
```

Comment:
@maxandersen know my favorite word to use instead of `up to now`, but I have yet to convince anyone in the tech community to use it :)

Reply:
What's the motivation for removing the reference to the curl appendix? I agree that it's *probably* not needed, but people seem to hit so many problems, and some flavours of linux are missing it. It's an innocuous change, but I'd rather not mix it in with the OpenAI content.